arches_lingo/src/arches_lingo/components/concept/ConceptImages/components/ConceptImagesViewer.vue

```diff
@@ -0,0 +1,195 @@
+            severity: SECONDARY,
+            outlined: true,
+        },
+        acceptProps: {
+            label: $gettext("Delete"),
+            severity: DANGER,
+        },
+    });
+}
+
+const openEditor = inject<(componentName: string) => void>("openEditor");
+async function getConceptImageResource(resourceInstanceId: string) {
+    try {
+        return await fetchLingoResource(
+            "digital_object_rdm_system",
+            resourceInstanceId as string,
+        );
+    } catch (error) {
+        console.error(error);
+    }
+}
+</script>
+
+<template>
+    <div class="section-header">
+        <h2>{{ props.sectionTitle }}</h2>
+        <Button
+            :label="$gettext('Add New Concept Image')"
+            @click="openEditor!(props.componentName)"
+        ></Button>
+    </div>
+    <div class="conceptImages">
+        <div
+            v-for="resource in resources"
+            :key="resource.resourceinstanceid"
+            class="conceptImage"
+        >
+            <div class="header">
+                <label
+                    for="conceptImage"
+                    class="text"
+                >
+                    <NonLocalizedStringWidget
+                        node-alias="name_content"
+                        graph-slug="digital_object_rdm_system"
+                        :mode="VIEW"
+                        :initial-value="
+                            resource.aliased_data.name.aliased_data.name_content
+                        "
+                    />
+                </label>
+                <div class="buttons">
+                    <Button
+                        icon="pi pi-file-edit"
+                        @click="openEditor!(props.componentName)"
+                    />
+                    <Button
+                        icon="pi pi-trash"
+                        :aria-label="$gettext('delete')"
+                        severity="danger"
+                        outlined
+                        @click="confirmDelete()"
+                    />
+                </div>
+            </div>
+            <FileListWidget
+                node-alias="content"
+                graph-slug="digital_object_rdm_system"
+                :initial-value="
+                    resource.aliased_data.content?.aliased_data.content
+                "
+                :mode="VIEW"
+                class="conceptImage"
+            />
+            <div class="footer">
+                <NonLocalizedStringWidget
+                    node-alias="statement_content"
+                    graph-slug="digital_object_rdm_system"
+                    :mode="VIEW"
+                    :initial-value="
+                        resource.aliased_data.statement?.aliased_data
+                            .statement_content
+                    "
+                />
+                {{
+                    resource.aliased_data.statement?.aliased_data
+                        .statement_content
+                }}
```

Comment:
nit: mind wrapping this in a span? no loose text should be added to the styleguide :D

Reply:
this should actually be purged; it's duplicative.